Data Structures and Algorithm (DSA)/readme.md

```diff
@@ -218,6 +219,55 @@ To the Point, Short and Practical - These 3 terms defines our Content
 </tr>
 </table>
 
+### Algorithms by Languages
+
+<table width="100%" id="algorithms-by-languages">
+<tr>
+<th>JavaScript</th>
+
+</tr>
+<tr>
+<td><a href="https://www.youtube.com/watch?v=P6XGSKO2RzI&t=519s">Quick Sort</a></td>
+
+</tr>
+<tr>
+<td><a href="https://www.youtube.com/watch?v=x_Z9FcAPmbk&t=1s">Merge Sort</a></td>
+</tr>
+<tr>
+<td><a href="https://www.youtube.com/watch?v=75jGy1xAhhs&t=2s">Binary Search</a></td>
+</tr>
+<tr>
+<th>Java</th>
+
+</tr>
+<tr>
+<td><a href="https://www.youtube.com/watch?v=h8eyY7dIiN4&t=2s">Quick Sort</a></td>
+
+</tr>
+<tr>
+<td><a href="https://www.youtube.com/watch?v=bOk35XmHPKs&t=1s">Merge Sort</a></td>
+</tr>
+<tr>
+<td><a href="https://www.youtube.com/watch?v=NFhOrxtXXcM&t=1s">Binary Search</a></td>
+</tr>
+<tr>
+<th>Python</th>
+
+</tr>
+<tr>
+<td><a href="https://www.youtube.com/watch?v=kFeXwkgnQ9U&t=1s">Quick Sort</a></td>
+
+</tr>
+<tr>
+<td><a href="https://www.youtube.com/watch?v=cVZMah9kEjI&t=1s">Merge Sort</a></td>
+</tr>
+<tr>
+<td><a href="https://www.youtube.com/watch?v=cVZMah9kEjI&t=1s">Binary Search</a></td>
+</tr>
+
+
+</table>
```

Comment:
You forgot to add a description in each resource. Check this [link](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md#how-to-add-a-resource)